testsuite/oauth/authorize/base.py

```diff
@@ -143,3 +174,5 @@ def find(self, selector):
         """ Shorthand for jQuery style element selecting.
         """
         return self.browser.find_element_by_css_selector(selector)
+
+class Authorizer(AbstractAuthorizer): pass
```

Comment:
It looks like you could just get rid of the Abstract Authorizer, move this module up a level, and have an Authorizer class that everything uses. NoneAuthorizer can inherit a concrete class just as well.